### PR TITLE
Replaced javax with jakarta API packages and fixed build.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Temporary build changes are in a separate file to be simply removed in the future.

### CODE REVIEW

File: docker/build_jakartaeetck.sh [hunk 1]
```diff
@@ -94,7 +94,7 @@ wget --progress=bar:force --no-cache $GF_BUNDLE_URL -O latest-glassfish.zip
 unzip -o latest-glassfish.zip
 ls -l $GF_HOME/glassfish5/glassfish/
 
-#temporary fix to get build passing until we have glassfish with new api jars
+# temporary fix to get build passing until we have glassfish with new api jars
```
Comment (jansupol) [inline docker/build_jakartaeetck.sh:97]: Can you update the copyright year in the sh files, too?

File: src/com/sun/ts/tests/jaxws/api/javax_xml_ws/WebServiceContext/WSWebServiceContext_wsappclient_vehicle_client.jar.sun-application-client.xml [hunk 2]
```diff
@@ -28,11 +28,11 @@
         <localpart>HelloPort</localpart>
       </wsdl-port>
       <stub-property>
-        <name>javax.xml.ws.security.auth.password</name>
+        <name>jakarta.xml.ws.security.auth.password</name>
```
Comment (jansupol) [inline src/com/sun/ts/tests/jaxws/api/javax_xml_ws/WebServiceContext/WSWebServiceContext_wsappclient_vehicle_client.jar.sun-application-client.xml:31]: Can you update the copyright year in the xml files, too?

File: docker/build_jakartaeetck.sh (hunk 1 above)
Comment (Tomas-Kraus) [reply docker/build_jakartaeetck.sh:97]: Done.

File: src/com/sun/ts/tests/jaxws/api/javax_xml_ws/WebServiceContext/WSWebServiceContext_wsappclient_vehicle_client.jar.sun-application-client.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply src/com/sun/ts/tests/jaxws/api/javax_xml_ws/WebServiceContext/WSWebServiceContext_wsappclient_vehicle_client.jar.sun-application-client.xml:31]: Done.

**Comment by scottmarlow** [comment]
Thanks for the change @Tomas-Kraus!  Thanks @jansupol + @alwin-joseph for reviewing! :)

**Pull request merged**